src/microbe_stage/PatchMap.cs

```diff
@@ -201,6 +201,18 @@ public void UpdateGlobalPopulations()
         }
     }
 
+    public long GetSpeciesPopulation(Species species)
```

Comment:
```suggestion
    public long GetSpeciesGlobalPopulation(Species species)
```
just to be extra clear what this returns and to not have a danger of mixing up the patch map and individual patch objects.